### PR TITLE
Added pools styling

### PR DESCRIPTION
**Pull request by vkulinich-cl**
Added components:

- Pool card
- Add Liquidity Modal
- Remove Liquidity Modal

### CODE REVIEW

File: src/sections/pools/modals/AddLiquidity/AddLiquidity.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+            align: "center",
+            mt: 20,
+            mb: 37,
+          }}
+        >
+          <Text color="basic300" fs={14}>
+            {t("pools.pool.liquidity.poolFees")}
+          </Text>
+          <Text fs={14} color="white">
+            0.15%
```
Comment (obsivac-cleevio) [inline src/sections/pools/modals/AddLiquidity/AddLiquidity.tsx:90]: radsi napsat TODO misto hodnoty, at se na to nezapomene

File: src/sections/pools/modals/AddLiquidity/AddLiquidity.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+          </Text>
+        </div>
+
+        <Text color="pink500" fs={15} font="FontOver" tTransform="uppercase">
+          Position Details
```
Comment (obsivac-cleevio) [inline src/sections/pools/modals/AddLiquidity/AddLiquidity.tsx:95]: do `translations`

**Review by obsivac-cleevio — CHANGES_REQUESTED** [review]

File: src/components/Input/Input.styled.ts [hunk 3]
```diff
@@ -45,7 +46,7 @@ export const SInput = styled.input<{ error?: string; unit?: string }>`
   padding: 20px 18px;
 
   ::placeholder {
-    color: rgba(${theme.colors.white}, 0.4);
+    color: rgba(114, 131, 165, 0.6);
```
[the remark this answers is not in the document]
Comment (vkulinich-cl) [reply src/components/Input/Input.styled.ts:49]: nie, to nema ani nazev konstanty ve figme, proto jsem nechtel vymyslet novou barvu

File: src/components/Slider/Slider.styled.ts [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  offset: number
+}>`
+  position: absolute;
+  ${({ row }) => (row === "top" ? `top: 0;` : `bottom: 0;`)};
+  ${({ offset }) => `left: ${offset}px;`};
+
+  height: 3px;
+  width: 1px;
+
+  background-color: rgba(150, 138, 158, 0.24);
```
[the remark this answers is not in the document]
Comment (vkulinich-cl) [reply src/components/Slider/Slider.styled.ts:95]: tady to stejny

**Review by obsivac-cleevio — APPROVED** [review]

**Pull request merged**